k8s/namespaces/em/em-showcase/em-showcase.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     nodejs:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/em/showcase:prod-98242c5-20220517091742 #{"$imagepolicy": "flux-system:em-showcase"}
+      image: hmctspublic.azurecr.io/em/showcase:prod-43a10a8-20220428104213 #{"$imagepolicy": "flux-system:em-showcase"}
```

Comment:
Please watch `imagePolicy `will update to latest available prod-* image

Reply:
@thomast1906 is that okay with you? Can I resolve this conversation?